test/tests/itemPaneTest.js

```diff
@@ -344,7 +373,479 @@ describe("Item pane", function () {
 				'1'
 			);
 		});
-	})
+	});
+
+	describe("Attachments pane", function () {
+		let paneID = "attachments";
+
+		beforeEach(function () {
+			Zotero.Prefs.set("panes.attachments.open", true);
+			Zotero.Prefs.set("showAttachmentPreview", true);
+			Zotero_Tabs.select("zotero-pane");
+		});
+
+		afterEach(function () {
+			Zotero_Tabs.select("zotero-pane");
+		});
+
+		it("should show attachments pane in library for regular item", async function () {
+			// Regular item: show
+			let attachmentsBox = ZoteroPane.itemPane._itemDetails.getPane(paneID);
+			let item = new Zotero.Item('book');
+			await item.saveTx();
+			await ZoteroPane.selectItem(item.id);
+			assert.isFalse(attachmentsBox.hidden);
+
+			// Child attachment: hide
+			let file = getTestDataDirectory();
+			file.append('test.pdf');
+			let attachment = await Zotero.Attachments.importFromFile({
+				file,
+				parentItemID: item.id
+			});
+			await ZoteroPane.selectItem(attachment.id);
+			assert.isTrue(attachmentsBox.hidden);
+
+			// Standalone attachment: hide
+			let attachment1 = await importFileAttachment('test.pdf');
+			await ZoteroPane.selectItem(attachment1.id);
+			assert.isTrue(attachmentsBox.hidden);
+		});
+
+		it("should not show attachments pane preview in reader best-matched attachment item", async function () {
```

Comment:
What does this mean?

Reply:
it's `should show ... in reader (when the attachment item is the best-matched attachment item of its parent item)`